view/template/nav/_footer.php

```diff
@@ -1,36 +1,51 @@
-<?php if (!defined('FOOTER_RENDERED')): ?>
-  <?php define('FOOTER_RENDERED', true) ?>
-  <div class="footer">
-    <?php if ($showLearnFooter): ?>
-      <?php echo View::render('nav/_learnFooter') ?>
-    <?php endif ?>
-    <div class="footer-standard <?php echo $isDark ? 'footer-standard--dark' : 'footer-standard--light' ?> <?php echo isset($isBordered) && !$isBordered ? 'footer-noborder' : '' ?>">
-      <nav class="control-group">
-        <div class="control-item">
-          <a href="/"><?php echo __('nav.home') ?></a>
-        </div>
-        <?php echo View::render('nav/_globalItems') ?>
-        <div class="control-item no-label-desktop">
-        <a href="https://t.me/lbryofficial" target="_blank"><span class="btn-label">Telegram</span> <i class="fab fa-telegram"></i></a>
-        </div>
-        <div class="control-item no-label-desktop">
-        <a href="https://www.instagram.com/lbryio/" target="_blank"><span class="btn-label">Instagram</span> <i class="fab fa-instagram"></i></a>
-        </div>
-        <div class="control-item no-label-desktop">
-        <a href="mailto:hello@lbry.io" target="_blank"><span class="btn-label">Contact Us</span> <span class="icon-fw icon-envelope"></span></a>
-        </div>
-        <div class="control-item no-label-desktop">
-        <a href="https://shop.lbry.io" target="_blank"><span class="btn-label">LBRY Shop</span> <span class="icon-fw icon-shopping-cart"></span></a>
-        </div>
-        <div class="control-item">
-          <a href="https://en.wikipedia.org/wiki/Free_Speech_Flag" class="footer-img-link">
-            <img src="/img/Free-speech-flag.svg" alt="Free Speech Flag" height="30"/>
-          </a>
-        </div>
-      </nav>
-      <div>
-        <a href="/join-us">LBRY is hiring!</a>
-      </div>
-    </div>
+<footer class="footer">
+  <div class="inner-wrap">
```

Comment:
Why use `.inner-wrap` repeatedly inside of elements instead of just naming this `.footer__inner`? 

(Analogous question for several other areas.)

Reply:
This enables us to have divs (like the footer) to have a background that extends the width of your screen, while also keeping a container. `.inner-wrap` is a global class so that's why you see it everywhere.